docs/ManualDeploymentGuideScalarDBServerOnAKS.md

```diff
@@ -1,72 +1,55 @@
+ScalarDB supports [several databases](https://github.com/scalar-labs/scalardb/blob/master/docs/scalardb-supported-databases.md). You need to prepare a database before you deploy ScalarDB Server. See [Set up a database for ScalarDB/ScalarDL deployment in Azure](./SetupDatabaseForAzure.md) for more details.
 
 ## Step 4. Create a bastion server
 
-For executing some tools to deploy and manage Scalar DB Server on AKS, you need to prepare a bastion server in the same VNet of the AKS cluster you created in **Step 3**. Please refer to the following document for more details.  
-
-* [Create a bastion server]() // TODO: Create a new document
+For executing some tools to deploy and manage ScalarDB Server on AKS, you need to prepare a bastion server in the same VNet of the AKS cluster you created in **Step 2**. See [Create a bastion server](./CreateBastionServer.md) for more details.
 
 ## Step 5. Prepare a custom values file of Helm
 
-You need to configure a custom values file for the Helm Chart of Scalar DB Server based on your environment (e.g., access information of the database you created in **Step 2**). Please refer to the following document for more details.  
-
-* [Configure a custom values file of Scalar Helm Chart]() // TODO: Create a new document in the Scalar Helm Chart repository
-
-## Step 6. Deploy Scalar DB Server using Scalar Helm Chart
+You need to configure a custom values file for the Helm Chart of ScalarDB Server based on your environment (e.g., access information of the database you created in **Step 3**). See [Configure a custom values file of Scalar Helm Chart](https://github.com/scalar-labs/helm-charts/blob/main/docs/configure-custom-values-file.md) for more details. 
 
-Deploy Scalar DB Server on your AKS cluster using Scalar Helm Chart. Please refer to the following document for more details.  
+Note: When you deploy your application on other environments than the AKS cluster that you deploy ScalarDB Server, you have to set the `LoadBalancer` to the `envoy.service.type` parameter to access Scalar Envoy from your application.
 
-* [Deploy Scalar Products using Scalar Helm Chart]() // TODO: Create a new document in the Scalar Helm Chart repository
+## Step 6. Deploy ScalarDB Server using Scalar Helm Chart
 
-## Step 7. Check the status of Scalar DB Server deployment
+Deploy ScalarDB Server on your AKS cluster using Scalar Helm Chart. See [Deploy Scalar Products using Scalar Helm Chart](https://github.com/scalar-labs/helm-charts/blob/main/docs/how-to-deploy-scalar-products.md) for more details.
 
-After deploying Scalar DB Server on your AKS cluster, you need to check the status of each component. Please refer to the following document for more details.  
+Note: We recommend creating a dedicated namespace by using the `kubectl create ns scalardb` command and deploying ScalarDB Server in the namespace by using the `-n scalardb` option with the `helm install` command.
```

Comment:
I added a new recommendation based on the offline discussion. We recommend using the dedicated k8s namespace for Scalar product deployment.